src/lizaalert/quizzes/validators.py

```diff
@@ -18,12 +18,12 @@ class Config:
 
 class AnswerModel(BaseModel):
     question_id: int
-    answer_id: List[int]
+    answer_id: List[Union[int, str]]
```

Comment:
В питоне >= 3.10 можно делать вот так
```suggestion
    answer_id: List[int | str]
```